secrets/config/clusters/uwhackweeks.cluster.yaml

```diff
@@ -1,36 +1,34 @@
-grafana_token: ENC[AES256_GCM,data:MbF6Wn7IJbbCjAraido0fWV8035wzNoslDzPyfju6kax69wfZYZZHNC6btqbF8Zloqo0kNZOjhxlSjtcI3qdgMuGFNXY4diIUHLguhTN0qrIok0F,iv:LqkWTZGgwBETfb35BSEroREtssCaitFqPR+jRLdzW6g=,tag:4u3G0CTQzeekaDsDI6BDZQ==,type:str]
+grafana_token: ENC[AES256_GCM,data:txwzDKVhrhxn01yVOMnvWWFP+7cSD6S790Y4dwrisenaKmaO9d690/ve9no/WZVTx9n2az3ZnyiWJ+L6zqt1fJNHMYY8/Wxsn1PCn1jssAGjAAe2,iv:I0Prnq5nMPvymUPv3c7galJ0q/mHeey+Q0QXawoWhMY=,tag:zySLCko+7QZq9vOBkF3Q7g==,type:str]
```

Comment:
Yeah, basically I've been thinking that the uwhackweeks hub was misfigured but I couldn't see the changed oauth_callback_url for the prod hub. Turned out Yuvi had put it in the secrets file so that it could be applied with the client_id and client_secret. Since we were extracting helm values, I alextracted the callback URL into its own prod.values.yaml file for explicitness and to put my mind at rest that indeed, uwhackweeks was configured correctly.

Reply:
uwhackweeks should now follow the same pattern as pangeo-hubs as those 2 are both using GitHub Teams Auth